storage/disk/partition_test.go

```diff
@@ -18,11 +18,13 @@ func TestPartitionTrie(t *testing.T) {
 		storage.MustParsePath("/foo/bar"),
 		storage.MustParsePath("/foo/baz/qux"),
 		storage.MustParsePath("/corge"),
+		storage.MustParsePath("/tenants/*/bindings"), // wildcard in the middle
+		storage.MustParsePath("/users/*"),            // wildcard at the end
```

Comment:
Hadn't thought about this case but it makes sense!

```
/users/a/a => row 1
/users/a/b => row 2
...
/users/a/<n> => row n
/users/b/a => row n+1
/users/b/b => row n+2
```

Can we have multiple wildcards? E.g., `/users/*/*`


```
/users/a/a/a => row 1
/users/a/a/b => row 2
/users/a/a/<n> => row n
/users/a/b/a => row n+1
...
/users/a/b/<m> => row n+m+1
```

Reply:
> Can we have multiple wildcards?

Yup that should be fine. I'll get some tests set up.